mock/cpp_mock_scenarios/src/metrics/out_of_range.cpp

```diff
@@ -30,10 +29,7 @@ class OutOfRangeScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit OutOfRangeScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "out_of_range",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.